src/imageupload.js

```diff
@@ -23,7 +24,7 @@ export default class ImageUpload extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ ImageUploadEngine ];
+		return [ ImageUploadEngine, ImageUploadProgress ];
```

Comment:
And of course both `ImageUploadButton` and `ImageUploadProgress` should require `ImageUploadEngine`.